pkg/kvstore/allocator/allocator.go

```diff
@@ -164,6 +164,10 @@ type Allocator struct {
 	// allocate an ID greater than this value.
 	max ID
 
+	// prefixMask if set, will be ORed to all selected IDs prior to
```

Comment:
If this is ORed into every ID, then it's not a mask (masks are usually ANDed). Why not just `prefix`?
Same comment for `WithPrefixMask` below.

Reply:
I'm using mask as the entire ID is ORed so the ID really must be in a mask as in https://en.wikipedia.org/wiki/Mask_(computing)

I don't mind using just prefix but I'm not sure it's any clearer.

Reply:
ok.